app/lib/common/providers/space_providers.dart

```diff
@@ -233,38 +228,27 @@ final spaceRelationsOverviewProvider =
   if (relatedSpaces == null) {
     throw SpaceNotFound;
   }
-  final membership = await ref.watch(roomMembershipProvider(spaceId).future);
-  bool hasMoreSubspaces = false;
-  bool hasMoreChats = false;
-  final List<Space> knownSubspaces = [];
-  final List<Convo> knownChats = [];
+  bool hasMore = false;
+  final List<String> knownSubspaces = [];
+  final List<String> knownChats = [];
   final List<String> suggested = [];
```

Comment:
this is an important simplification for performance: by using the strings and just relying on the underlying rust having checked properly, we can remove an entire ref.watch call stack and the necessary memory problems.